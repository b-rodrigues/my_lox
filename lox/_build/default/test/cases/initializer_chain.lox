class Base { init(a) { this.a = a; } show() { return "Base(" + this.a + ")"; } }
class Sub < Base {
  init(a, b) { super.init(a); this.b = b; }
  show() { return "Sub(" + this.a + "," + this.b + ")"; }
}
var s = Sub(10, 5);
print s.show();