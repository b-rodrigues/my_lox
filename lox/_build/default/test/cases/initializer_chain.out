Sub(10,5)
